client/client.js

```diff
@@ -60,6 +60,7 @@ function CloudCmdProto(DOM) {
     this.prefixURL = '';
     
     this.MIN_ONE_PANEL_WIDTH = 1155;
+    this.MOBILE_ONE_PANEL_WIDTH = 600;
```

Comment:
We can use css variables here, something like `—-is-mobile` [query.css](https://github.com/coderaiser/cloudcmd/blob/1a0af863a21e6f9247c9310621f6b7501555ec41/css/query.css#L117)

https://stackoverflow.com/questions/41725725/access-css-variable-from-javascript

Reply:
I can look into it , but i think this improvement should be done in other PR and also change/fix the existing MIN_ONE_PANEL_WIDTH so we the code wont have mix and match code.

Let me know if i should change only for the new const (MOBILE_ONE_PANEL_WIDTH)

Reply:
Yes, we don’t need this constant, since screen size is checked with media queries, so we just set a flag in css, and then check it in js

Reply:
Not sure i understand your answer.

You mean i should query the new value (600) from css variable and leave the existing (MIN_ONE_PANEL_WIDTH ) const for now ?

Reply:
Yes, let’s keep it for now, we can make this in separate PR